esileapclient/common/utils.py

```diff
@@ -0,0 +1,56 @@
+import re
+import operator
+
+OPS = {
+    '>=': operator.ge,
+    '<=': operator.le,
+    '>': operator.gt,
+    '<': operator.lt,
+    '=': operator.eq,
+}
+
+
+def parse_property_filter(filter_str):
+    """Parse a property filter string into a key, operator, and value."""
+    match = re.match(r'([^><=]+)([><=]+)(.+)', filter_str)
```

Comment:
I think we should be building this up from the list of operations in `OPS`, so that if we add new operations we don't also need to modify the match expression.